src/JLSOFile.jl

```diff
@@ -1,3 +1,5 @@
+import Base.Threads: @spawn
```

Comment:
```suggestion
using Base.Threads: @spawn
```

Prefer `using` over `import.
Per https://github.com/invenia/BlueStyle#module-imports